src/System.Runtime.Environment/src/RuntimeInfo.Linux.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace System.Runtime.InteropServices
+{
+    public static class RuntimeInfo
```

Comment:
There is an existing type called `RuntimeEnvironment` which is a `class` and not a part of CoreFX (at least the portable subset).